test/bigtest/network/configs/interfaces.js

```diff
@@ -0,0 +1,18 @@
+import { noop } from 'lodash';
+import { INTERFACES_API } from '../../../../src/common/constants';
+
+const configInterfaces = server => {
+  server.get(INTERFACES_API, (schema) => {
+    return schema.interfaces.all();
+  });
+
+  server.get(`${INTERFACES_API}/:id`, (schema, request) => {
+    return schema.interfaces.find(request.params.id).attrs;
```

Comment:
what if wrong `id` is provided?

Reply:
I've made some changes, do you mean something like that?

Reply:
I've just asked a question, I'm not sure that your changes are answer to it